web/tendrl/multiuser_cluster_import.rst

```diff
@@ -1,2 +1,52 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+
```

Comment:
Could you specify if this test case is positive or negative? Adding either `caseposneg: positive` or `caseposneg: positive`.